src/main/java/pages/cucumber/CucumberToolsPage.java

```diff
@@ -0,0 +1,25 @@
+package pages.cucumber;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pages.AbstractPage;
+
+public class CucumberToolsPage extends AbstractPage {
+
+    @FindBy(xpath = "//h1[contains(text(),'Cucumber Open')]")
+    private WebElement actualPageTitle;
+
+    public CucumberToolsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public boolean isPageWithTitleDisplayed() {
+        return actualPageTitle.isDisplayed();
+    }
+
+    @Override
+    protected AbstractPage openPage() {
```

Comment:
```
protected CucumberToolsPage openPage() {
 return new CucumberToolsPage(driver);
}
```

Reply:
Refactored, thank you.
Please see in the following commit.